x/incentive/types/params.go

```diff
@@ -315,3 +329,29 @@ func validateEdenRewardApr(i interface{}) error {
 
 	return nil
 }
+
+func validateDistributionEpochLps(i interface{}) error {
+	v, ok := i.(int64)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	if v < 1 {
```

Comment:
can you update this to `v <= 0` to clearly express that it should be positive integer?